packages/framework/tinylicious-client/src/TinyliciousAudience.ts

```diff
@@ -3,18 +3,25 @@
  * Licensed under the MIT License.
  */
 
+import { assert } from "@fluidframework/common-utils";
 import { ServiceAudience } from "@fluidframework/fluid-static";
 import { IClient } from "@fluidframework/protocol-definitions";
-import { ITinyliciousAudience, TinyliciousMember } from "./interfaces";
+import { ITinyliciousAudience, TinyliciousMember, TinyliciousUser } from "./interfaces";
 
+/**
+ * {@inheritDoc ITinyliciousAudience}
+ */
 export class TinyliciousAudience extends ServiceAudience<TinyliciousMember> implements ITinyliciousAudience {
   /**
    * @internal
    */
   protected createServiceMember(audienceMember: IClient): TinyliciousMember {
+    const tinyliciousUser = audienceMember.user as TinyliciousUser;
+    assert(tinyliciousUser !== undefined, "Specified user was not of type \"TinyliciousUser\".");
```

Comment:
This is not a very strong type check - maybe that's OK?  Or should it be checking the `name` property is `typeof` to `string`?